docs/bosh-init-usage.md

```diff
@@ -20,38 +20,45 @@ name: bosh
 
 releases:
 - name: bosh
-  url: https://bosh.io/d/github.com/cloudfoundry/bosh?v=218
-  sha1: 7ad794897468a453e81b018da53c8475a23cef2b
+  url: https://bosh.io/d/github.com/cloudfoundry/bosh?v=219
+  sha1: bbd03790a2839aab26d3fa4cfe1493d361872f33
 - name: bosh-softlayer-cpi
-  url: file://./bosh-softlayer-cpi-0+dev.1.tgz #<--- Replace with SL eCPI release
+  url: file://./bosh-softlayer-cpi-0+dev.1.tgz
 
 resource_pools:
 - name: vms
   network: default
   stemcell:
-    url: file://./light-bosh-stemcell-3031-softlayer-esxi-ubuntu-trusty-go_agent.tgz # <--- Stemcell and bosh-agent should need some changes due to constrains of Softlayer
+    url: file://./light-bosh-stemcell-3031-softlayer-esxi-ubuntu-trusty-go_agent.tgz
   cloud_properties:
-    Domain: softlayer.com
-    VmNamePrefix: bosh-experimental  # <--- It is better to use a catchy name which will be used in the following section
-    StartCpus: 1
-    MaxMemory: 1024
+    Domain: SOFTLAYER.COM      # <--- domain name for new director
+    VmNamePrefix: VMNAMEPREFIX # <--- indicate a hostname prefix for new director
```

Comment:
I believe Alexey's changes completely changed the 'VmNamePrefix' property to 'Hostname'.